persistence/relational-jdbc/build.gradle.kts

```diff
@@ -35,6 +35,9 @@ dependencies {
 
   implementation(libs.smallrye.common.annotation) // @Identifier
   implementation(libs.postgresql)
+  implementation("software.amazon.awssdk:apache-client")
+  implementation(platform(libs.quarkus.amazon.services.bom))
+  implementation("io.quarkiverse.amazonservices:quarkus-amazon-rds")
```

Comment:
I guess these deps are not required for building the JDBC code itself. They probably need to be in `runtime/common`.

The idea is to keep the JDBC persistence lean, but add RDS where drivers would normally be added. WDYT?

(sorry, I missed this in my previous review)

Reply:
Note: `apache-client` is probably already present in "runtime" modules.